### PR TITLE
dashboard18-01-24 changes

### PR DESCRIPTION
**Pull request by Manikanta662001**

### CODE REVIEW

File: src/pages/dashboard/adminDashboard/AdminDashboard.tsx [hunk 1]
```diff
@@ -366,10 +367,17 @@ const AdminDashboard = () => {
     });
   };
   const handleSocket = () => {
-    socket.emit("testing", {
-      opponentId: currentUser._id,
-      sender: getFullName(currentUser),
+    // socket.emit("testing", {
```
Comment (ResourceOne2011) [inline src/pages/dashboard/adminDashboard/AdminDashboard.tsx:370]: remove the commented code

File: src/pages/dashboard/adminDashboard/AdminDashboard.tsx [hunk 2]
```diff
@@ -415,7 +423,7 @@ const AdminDashboard = () => {
       </div>
       <div className="admin-btns">
         <Button variant="danger" onClick={handleSocket}>
```
Comment (ResourceOne2011) [inline src/pages/dashboard/adminDashboard/AdminDashboard.tsx:421]: change the naming convention handleSocket to handleAdminMessage or handleAdminBroadcastMessage

Comment (Manikanta662001) [reply src/pages/dashboard/adminDashboard/AdminDashboard.tsx]: done

File: src/pages/dashboard/adminDashboard/AdminDashboard.tsx (hunk 1 above)
Comment (Manikanta662001) [reply src/pages/dashboard/adminDashboard/AdminDashboard.tsx]: done

File: src/pages/dashboard/clientDashboard/index.tsx [hunk 3]
```diff
@@ -1,31 +1,85 @@
...
 import { useLocation } from "react-router-dom";
 import "./index.css";
 import httpMethods from "../../../api/Service";
 import { TicketModal } from "../../../modals/TicketModals";
 import TaskTable, { TableHeaders } from "../../../utils/table/Table";
 import TicketsMain from "../../tickets/TicketsMain";
+import PieChartComponent from "../../../components/pieChart/PieChart";
 
 const ClientDashboard = () => {
   const { state } = useLocation();
```
Comment (ResourceOne2011) [inline src/pages/dashboard/clientDashboard/index.tsx:11]: dont render the state directly assign it to state varible like clientData, setClientData

Comment (Manikanta662001) [reply src/pages/dashboard/clientDashboard/index.tsx:11]: done

**Review by ResourceOne2011 — APPROVED** [review]

**Pull request merged**